tests/unittests/GraphOptzTest.cpp

```diff
@@ -726,10 +726,14 @@ TEST_F(GraphOptz, BatchNormAfterConvNotOptimizeForTrain) {
 
   EXPECT_EQ(F_->getNodes().size(), 3);
 
+  auto *clone = F_->clone(F_->getName().str() + "_clone");
+  ::glow::convertPlaceholdersToConstants(F_, bindings_, {});
+  optimizedF_ = optimizeFunction(clone);
```

Comment:
We need to optimize `optimizedF_` for training.
```suggestion
::glow::optimize(optimizedF_, CompilationMode::Train);
```